### PR TITLE
Add script for deploying stack hub

### PR DESCRIPTION
**Pull request by jgawor**
Add `hub_deploy.sh` script that can deploy the NGINX image along with the stack images into the custom registry and deploy the NGINX container into OpenShift cluster. This gives the ability to entirely clone all stack hub images into own registry and host the index in the cluster.

Also, update the `appsody_stack_config.yaml` to point to the latest stacks, fix some typos, rename `repo-index` to `stack-hub-index`, etc.

### CODE REVIEW

**Review by gmarcy — APPROVED** [review]
lgtm

File: scripts/hub_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    fi
+}
+
+image_mirror() {
+    local file=$1
+    oc image mirror -f "$file" --insecure
+}
+
+get_route() {
+    for i in 1 2 3 4 5 6 7 8 9 10; do
```
Comment (groeges) [inline scripts/hub_deploy.sh:33]: What is the purpose of this for loop as  the value of `i` isnt used?

Comment (jgawor) [reply scripts/hub_deploy.sh:33]: It's a silly loop to wait a few seconds. Sometimes it takes a bit for the route information (the endpoint) to be populated.

**Review by groeges — APPROVED** [review]
LGTM

**Comment by SueChaplain** [comment]
@groeges @jgawor - presumably I need to make changes to the stack hub documentation to capture the extra capability provided here?

See https://www-03preprod.ibm.com/support/knowledgecenter/en/SSCSJL_4.3.x/guides/creating-a-stack-hub/creating-a-stack-hub.html - we have a section for private repository that talks about NGINX. Please advise.

**Comment by SueChaplain** [comment]
To tie in - guide update: https://github.ibm.com/IBMCloudPak4Apps/icpa-guides-kc/pull/165

**Pull request merged**